modules/ROOT/pages/functions/string.adoc

```diff
@@ -148,6 +148,163 @@ RETURN ltrim('   hello')
 
 ======
 
+
+
+[[functions-normalize]]
+== normalize()
+
+`normalize()` returns the given `STRING` normalized using the `NFC` normalization form.
+
+The `normalize()` function is useful for converting `STRING` values into comparable forms.
+When comparing two `STRING` values, it is their Unicode codepoints that are compared.
+In Unicode, a codepoint for a character that looks the same may actually be represented by two, or more, different codepoints.
+For example, the character `<` can be represented as `\uFE64` or `\u003C`. Visually, the character may look the same,
+but if compared, Cypher will return false as `\uFE64` does not equal `\u003C`. Using the `normalize()` function one can
+normalize the codepoint `\uFE64` to `\u003C`, creating a single codepoint representation, allowing them to be successfully compared.
```

Comment:
Can we say in brackets which normalization form uses `\uFE64` and `\u003C`?

It feels like something is missing from this sentence: "but if compared, Cypher will return false as `\uFE64` does not equal `\u003C`." Is this because Cypher uses a particular normalization form? and, if so, should we mention what that is?

```suggestion
For example, the character `<` can be represented as `\uFE64` or `\u003C`. Visually, the character may look the same,
but if compared, Cypher will return `false` as `\uFE64` does not equal `\u003C`. Using the `normalize()` function, it is possible to
normalize the codepoint `\uFE64` to `\u003C`, creating a single codepoint representation, allowing them to be successfully compared.
```

Reply:
It is because \uFE64 != \u003C 😅 not due to Cypher, just because those codes don't match in any way, I guess what this paragraph is trying to explain is that just because a character looks like another, it doesn't mean the encoding is the same. One could think about it like 2 colours, if both are red, they might be different shades or red, our eyes may not pick that up, but the computer knows, as the computer can read the exact "wavelength" so to say, so the computer will say that they are not equal, normalization is a way to make the computer be less fussy and accept that red = red, no matter the shade :)

Reply:
That makes sense!